src/index.js

```diff
@@ -1,27 +1,37 @@
-const API_URL =  "https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/xHcNAgk6UkMi3fEdct8S/scores";
-
-const getData =  ()=>{
-  const response = fetch(API_URL, config);
-  let data = response.json(display);
-  display(data);
-  console.log(data);
-  console.log(data.result[0]);
-
+import './style.css';
+import { postMessage, API_URL } from './modules/submitScore';
+
+const getScores = () => {
+  const displayContent = document.getElementById('table_item');
+  const scoreList = document.createElement('li');
+  const config = {
+    headers: {
+      'Access-Control-Allow-Origin': origin,
+      Accept: 'application/json',
+    },
+  };
+  fetch(API_URL, config)
+    .then((response) => response.json())
+    .then((data) => {
+      for (let i = 0; i < data.result.length; i += 1) {
+        scoreList.textContent = `
+    ${data.result[i].user}
+    ${data.result[i].score}`;
+        displayContent.append(scoreList);
+      }
+    });
 };
 
-getData();
-
-
-const displayContainer = document.querySelector('.table_item');
-let feed = '';
-
-const display = (data) => {
-
-for(let i = 0; i <data.result.length; i++) {
-feed +=  `<tr><td>${data.result[i].user}</td><td>${data.result[i].score}</td></tr>`
-
-}
+getScores();
 
-displayContainer.innerHTML = feed;
+const refresh = document.getElementById('refresh');
+refresh.addEventListener('click', () => {
+  getScores();
+});
```

Comment:
- [ ] <b>REQUIRED:</b> As I mentioned in the review summary above. `refresh` feature is not working as expected. check out this [video](https://www.loom.com/share/d6c205de9c7943719e7eb7dee0586e04) for a clear image of what is happening.

Reply:
May I know what is the expected behavior? Your suggestions are vague and honestly you supposed to be explicit and helpfull with them. I think this is not the case.